qubes/vm/mix/net.py

```diff
@@ -128,6 +128,10 @@ def firewall_conf(self):
     # used in networked appvms or proxyvms (netvm is not None)
     #
 
+    @qubes.stateless_property
+    def visible_mac(self):
+        '''MAC address of this domain as seen by the domain.'''
+        return self.mac
```

Comment:
`visible_mac` property isn't needed at all, you can use `self.mac` directly in `create_qdb_entries()` below.

Reply:
Ok it was an anticipate code if we would need to do some tweaks here in the future.

Reply:
Better drop it for now. If we'll need to, it's easy enough to add it. But if we add it now, we'll need to keep it in this form. Which means if it turns out we'll something slightly different here, we'll need to introduce yet another property then.

Reply:
Done :)